tests/utils.go

```diff
@@ -1705,6 +1707,33 @@ func ExecuteCommandOnPodV2(virtCli kubecli.KubevirtClient, pod *k8sv1.Pod, conta
 	return stdoutBuf.String(), stderrBuf.String(), nil
 }
 
+func GetRunningVirtualMachineInstanceDomainXML(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) (string, error) {
```

Comment:
You should really rebase. That function existed already on master, I just modified it.